pyvcloud/vcd/vapp.py

```diff
@@ -242,3 +243,36 @@ def getVmFromVApp(self, vm_name):
                         if vm.get('name') == vm_name:
                             return vm
         raise Exception('can\'t find VM')
+
+    def add_disk_to_vm(self, vm_name, disk_size):
+        """
+        Add a virtual disk to a virtual machine in the vApp.
+        It assumes that the VM has already at least one virtual hard disk
+        and will attempt to create another one with similar characteristics.
+
+        :param vm_name: (str): The name of the vm to be customized.
+        :param disk_size: (int): The size of the disk to be added, in MBs.
+        :return:  A :class:`lxml.objectify.StringElement` object describing the asynchronous Task creating the disk.
+
+        :raises: Exception: If the named VM cannot be located or another error occured.
+        """  # NOQA
+        vm = self.get_vm(vm_name)
+        disk_list = self.client.get_resource(vm.get('href') + '/virtualHardwareSection/disks')  # NOQA
+        last_disk = None
+        for disk in disk_list.Item:
+            if disk['{' + NSMAP['rasd'] + '}Description'] == 'Hard disk':  # NOQA
+                last_disk = disk
+        assert last_disk is not None
+        new_disk = deepcopy(last_disk)
+        addr = int(str(last_disk['{' + NSMAP['rasd'] + '}AddressOnParent'])) + 1  # NOQA
+        instance_id = int(str(last_disk['{' + NSMAP['rasd'] + '}InstanceID'])) + 1  # NOQA
+        new_disk['{' + NSMAP['rasd'] + '}AddressOnParent'] = addr
+        new_disk['{' + NSMAP['rasd'] + '}ElementName'] = 'Hard disk %s' % addr  # NOQA
+        new_disk['{' + NSMAP['rasd'] + '}InstanceID'] = instance_id
+        new_disk['{' + NSMAP['rasd'] + '}VirtualQuantity'] = disk_size * 1024 * 1024  # NOQA
+        new_disk['{' + NSMAP['rasd'] + '}HostResource'].set('{http://www.vmware.com/vcloud/v1.5}capacity', str(disk_size))  # NOQA
```

Comment:
http://www.vmware.com/vcloud/v1.5 should be replaced with NSMAP['vcloud']

Reply:
Otherwise LGTM

Reply:
I made the change